experimental/dds/tree2/src/test/feature-libraries/sequence-field/sequenceChangeRebaser.spec.ts

```diff
@@ -55,7 +55,6 @@ function generateLineage(
 }
 
 const testChanges: [string, (index: number, maxIndex: number) => SF.Changeset<TestChange>][] = [
-	["SetValue", (i) => Change.modify(i, TestChange.mint([], 1))],
```

Comment:
This effectively tests nested changes, so we want to keep it. Just rename it to "NestedChange".